### PR TITLE
[Windows] Fix properties view & disable tree and table lines

### LINKED ISSUE #137: Missing styles for table borders and properties view background
Hi, I was looking for an alternative to Darkest Dark (because of it being very heavy with additional stuff I don't want and commercial licensing) and found your theme. Unfortunately there are some glaring white spots. Is there a chance these can be fixed? I know some controls are not themeable on Windows though there have been some improvements in the newer Eclipse versions (like the dark scrollbar). Darkest Dark seems to have no problems at all with these though (possibly using some more aggressive workarounds).

Environment:

- Eclipse IDE for Java Developers 2020-09 (4.17.0)
- Windows 10 2004
- [SapMachine 11.0.8 JDK](https://sap.github.io/SapMachine/)
- [ABAP Development Tools for Eclipse (ADT) 3.12.5](https://tools.hana.ondemand.com/#abap)
- Spectrum Theme 1.0.0.202009070230 installed using the marketplace
- This is a fresh Eclipse installation and workspace

### Table views have white borders

image

image

image

The default dark theme has these as well:

image

Darkest Dark has solved this somehow.

### Properties view has a white background

image

image

With default dark theme:

image

The code itself is also entirely not color themed but I am fairly certain that's because the ABAP Development Tools are using their own color variables...

I looked through the windows tag but have not seen these issues reported yet.

### PR DESCRIPTION
**Pull request by AObuchow**
Fix #137

Signed-off-by: Andrew Obuchowicz <andrew@aobuchow.com>

### CODE REVIEW

**Comment by AObuchow** [comment]
@flaiker I worked on a fix yesterday night finally for the bug you reported. Do you mind installing an [integration build of Eclipse](https://download.eclipse.org/eclipse/downloads/) and installing spectrum theme using the following update site?
Update site: `https://raw.githubusercontent.com/AObuchow/Eclipse-Spectrum-Theme/builds-properties_view_fix/updatesite/`

If you encounter errors installing it, make sure you uninstall spectrum theme first if already installed (I just encountered this issue myself).

File: com.aobuchow.themes.spectrum/css/default_overrides.css [hunk 1]
```diff
@@ -359,3 +359,27 @@ ProgressIndicator {
...
+
+
+/* ############################## Issue 137  ############################## */
+
+TabbedPropertyTitle > CLabel {
+	color: white;
+}
+
+TabbedPropertyTitle {
+	swt-backgroundGradientStart-color:   #2f2f2f;
```
Comment (AObuchow) [inline com.aobuchow.themes.spectrum/css/default_overrides.css:371]: @PyvesB I was having trouble setting these colors to color extension points (eg. `'#com-aobuchow-themes-spectrum-BASE_COLOR'`). Have you encountered a similar issue with your theme by any chance?

**Comment by flaiker** [comment]
The issues are fixed! Thank you so much.

<details><summary>Screenshots</summary>

### Table views

image

image

### Properties view

image

image

</details>

File: com.aobuchow.themes.spectrum/css/default_overrides.css (hunk 1 above)
Comment (PyvesB) [reply com.aobuchow.themes.spectrum/css/default_overrides.css:371]: https://github.com/PyvesB/eclipse-planet-themes/blob/cd9e3d404e217198a6ffad1f4ce9fdeda73de307/planet-themes-plugin/css/planets.css#L95

^ I've tended to only add properties when I've spotted an explicit use-case, so I'm assuming that works. The casing of the property name is different from yours though, not sure whether it matters.

**Pull request merged**